htmd/builder/amber.py

```diff
@@ -131,33 +138,14 @@ def listFiles():
 def _locateFile(fname, type, teleap):
```

Comment:
sorry, I'll have to ask for a tiny change :P `type` is a python keyword. Try not to use it as a variable (might have been me who did it first but anyway)

Reply:
We were thinking the same at the same time and already pushing a fix. Done.